docs/src/usage.md

```diff
@@ -23,6 +23,8 @@ This works like `cargo test` except that it will analyze all proof harnesses ins
 
 Common to both `kani` and `cargo kani` are many command-line flags:
 
+ * `--concrete-playback=JustPrint`: Generates a Rust unit test case that plays back a failing proof harness using a concrete counterexample. The additional `--concrete-playback=InPlace` flag will automatically add this unit test to the source code next to the proof harness.
```

Comment:
`--concrete-playback=[print | inplace]`: Generates a Rust unit test that represents a concrete counter-example found by Kani. Use `print` argument to print the test in the terminal or `inplace` to automatically add this unit test to the source code next to the target proof harness.

Reply:
done